graph/is_graph_bipartite.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ *  @file is_graph_bipartite
+ *
+ *	@brief Algorithm to check whether a graph is [bipartite](https://en.wikipedia.org/wiki/Bipartite_graph)
+ *	
+ *	@details
+ *	A graph is a collection of nodes also called vertices and these vertices
+ *	are connected by edges.A bipartite graph is a graph whose vertices can be
+ * 	divided into two disjoint and independent sets U and V such that every edge 
+ *	connects a vertex in U to one in V. 	
+ *	
+ *	The given Algorithm will determine whether the given graph is bipartite or not
+ *	
+ *	
+ * 	Example - Here is a graph g1 with 5 vertices and is bipartite
+ *	
```

Comment:
This will not be displayed correctly in the online documentation. Please refer to other files in the folder [like this](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/graph/lowest_common_ancestor.cpp) for reference to how to display a graph.